workflow/scripts/osemosys_global/powerplant/investment_constraints.py

```diff
@@ -1,17 +1,113 @@
-"""Function to set min and max capacity investment constraints."""
-
+"""Functions to set powerplant investment constraints."""
 import pandas as pd
+import itertools
 
-from data import get_years
+from data import(
+    get_years,
+    get_max_value_per_technology
+    )
 
 from utils import apply_dtypes
 
+def set_renewable_limits(res_limits, tech_code_dict,
+                         custom_nodes, custom_nodes_res_limits,
+                         residual_capacity, start_year, 
+                         end_year, region_name):
```

Comment:
This function is just moved from the old `max_capacity` file, correct? Im pretty sure it is so I only skimmed the implementation since it worked before! If this is new code though and requires a more in depth review just let me know!

Reply:
Correct!